images/linux/scripts/installers/action-archive-cache.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+################################################################################
+##  File:  action-archive-cache.sh
+##  Desc:  Download latest release from https://github.com/actions/action_verions
+##         and un-tar to /opt/actionarchivecache
+################################################################################
+
+# Prepare directory and env variable for ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE
+ACTION_ARCHIVE_CACHE_DIR=/opt/actionarchivecache
+mkdir -p $ACTION_ARCHIVE_CACHE_DIR
+echo "ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE=$ACTION_ARCHIVE_CACHE_DIR" | tee -a /etc/environment
+chmod -R 777 $ACTION_ARCHIVE_CACHE_DIR
+
+# Download latest release from github.com/actions/action-versions and untar to /opt/actionarchivecache
+pushd $ACTION_ARCHIVE_CACHE_DIR
+latestReleaseName=$(curl -s -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" https://api.github.com/repos/actions/action-versions/releases/latest | jq --raw-output  '.name')
+curl -s -O -L https://github.com/actions/action-versions/releases/download/$latestReleaseName/action-versions.tar.gz
```

Comment:
downloading once may fail, I'd use the `download_with_retries` helper defined here:  https://github.com/actions/runner-images/blob/main/images/linux/scripts/helpers/install.sh